lightning/src/routing/scoring.rs

```diff
@@ -1247,6 +1269,13 @@ DirectedChannelLiquidity< L, HT, T> {
 			}
 		}
 
+		if score_params.probing_diversity_penalty_msat != 0 {
+			let time_since_update = last_duration_since_epoch.saturating_sub(*self.last_datapoint);
```

Comment:
Could use some explanation as to why `last_duration_since_epoch` is used here. Intuitively I expected to see the current time here minus `last_datapoint`.